source/databricks/calculation-engine/tests/integration/calculator/calculator_job/conftest.py

```diff
@@ -96,12 +114,68 @@ def test_data_written_to_delta_tables(
     )
 
 
+@pytest.fixture(scope="session")
+def charge_input_data_written_to_delta_tables(
+    spark: SparkSession,
+    test_files_folder_path: str,
+    input_database_created: None,
+    data_lake_path: str
+) -> None:
+    # Charge master data periods
+    spark.sql(
+        f"CREATE TABLE IF NOT EXISTS {infra.INPUT_DATABASE_NAME}.{infra.CHARGE_MASTER_DATA_PERIODS_TABLE_NAME} USING DELTA \
+            LOCATION '{data_lake_path}/calculation_input/charge_master_data_periods'"
+    )
+
+    charge_master_data_periods_df = spark.read.csv(
+        f"{test_files_folder_path}/ChargeMasterDataPeriods.csv",
+        header=True,
+        schema=charge_master_data_periods_schema,
+    )
+
+    charge_master_data_periods_df.write.format("delta").mode("overwrite").option("overwriteSchema", "true").saveAsTable(
+        f"{infra.INPUT_DATABASE_NAME}.{infra.CHARGE_MASTER_DATA_PERIODS_TABLE_NAME}"
+    )
+
+    # Charge link periods
+    spark.sql(
+        f"CREATE TABLE IF NOT EXISTS {infra.INPUT_DATABASE_NAME}.{infra.CHARGE_LINK_PERIODS_TABLE_NAME} USING DELTA \
+            LOCATION '{data_lake_path}/calculation_input/charge_link_periods'"
+    )
+
+    charge_links_periods_df = spark.read.csv(
+        f"{test_files_folder_path}/ChargeLinkPeriods.csv",
+        header=True,
+        schema=charge_link_periods_schema,
+    )
+
+    charge_links_periods_df.write.format("delta").mode("overwrite").option("overwriteSchema", "true").saveAsTable(
+        f"{infra.INPUT_DATABASE_NAME}.{infra.CHARGE_LINK_PERIODS_TABLE_NAME}"
+    )
+
+    # Charge price points
+    spark.sql(
+        f"CREATE TABLE IF NOT EXISTS {infra.INPUT_DATABASE_NAME}.{infra.CHARGE_PRICE_POINTS_TABLE_NAME} USING DELTA \
+            LOCATION '{data_lake_path}/calculation_input/charge_price_points'"
+    )
+
+    charge_prices_points_df = spark.read.csv(
+        f"{test_files_folder_path}/ChargePricePoints.csv",
+        header=True,
+        schema=charge_price_points_schema,
+    )
+
+    charge_prices_points_df.write.format("delta").mode("overwrite").option("overwriteSchema", "true").saveAsTable(
+        f"{infra.INPUT_DATABASE_NAME}.{infra.CHARGE_PRICE_POINTS_TABLE_NAME}"
+    )
+
+
 @pytest.fixture(scope="session")
 def executed_balance_fixing(
     spark: SparkSession,
     calculator_args_balance_fixing: CalculatorArgs,
     migrations_executed: None,
-    test_data_written_to_delta_tables: None,
+    metering_points_and_time_series_input_data_written_to_delta_tables: None,
```

Comment:
What do you think about this?
```suggestion
    energy_test_data_written_to_delta: None,
```